src/cloud/project/project-integrate-blackfire.md

```diff
@@ -0,0 +1,12 @@
+---
+group: cloud-guide
```

Comment:
I don't think you need `group` or `title`. Unless you think keeping `title` would be useful for future refrence.

You definitely don't need `functional_areas`.